docs/app-integration-development/11-OAuth-2-Id-token.md

```diff
@@ -74,7 +74,7 @@ The claims included in the PagerDuty ID Token are described here in detail:
  `auth_time`     |  Time when authentication occurred, in a Unix timestamp
  `iat`           |  Time when the ID Token was issued, in a Unix timestamp
  `purpose`       |  Purpose of the JWT. For PagerDuty ID tokens, this is always identification.
- `at_hash`       |  ID Token hash value
+ `at_hash`       |  Hash value of corresponding access token issued
```

Comment:
🤔 we could add a note about how the hash is created and could be verified? but i'm also fine with shipping as is!

Reply:
It's part of the OpenID spec so I'm not too worried here.